README.md

```diff
@@ -32,16 +32,17 @@ Demonstrate your understanding of this week's concepts by answering the followin
 
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your Team Lead
 
-1. If you were to describe semantic HTML to the next cohort of students, what would you say?
+1. If you were to describe semantic HTML to the next cohort of students, what would you say? Semantic HTML would be using semantic tags that let the developer or the browser know what its designating a certain styling. 
 
-2. Name two big differences between ```display: block;``` and ```display: inline;```.
+2. Name two big differences between ```display: block;``` and ```display: inline;```.Display block takes up the whole line and can not share space with another tag, display inline can share space unlike div tags, span tags are example of inline block elements. 
 
 3. What are the 4 areas of the box model?
-
-4. While using flexbox, what axis does the following property work on: ```align-items: center```?
+The four areas of the box model would be padding,margin,border and content. 
+4. While using flexbox, what axis does the following property work on: 
+```align-items: center```? The horizontal axis.
 
```

Comment:
On the cross-axis, the children elements are aligned together where the x and y axis meet so one won't be higher or lower than the other.
y
|            
item]---x-axis-->  <---item--->